.prettierignore

```diff
@@ -1,3 +1,5 @@
 docs/repositories.md
 docs/rules.md
 docs/toolchain.md
+# whitespsce matters: a regex is used to parse and automatically update this json
+.github/renovate-regex.json
```

Comment:
Those crucial single-backslashes get "cleaned out" if we let Prettier have free reign.